control-occplugin.sh

```diff
@@ -8,10 +8,10 @@ requires:
   - grpc
   - protobuf
   - "GCC-Toolchain:(?!osx)"
+  - InfoLogger
```

Comment:
couldn't you use libinfologger instead of infologger ?

Reply:
No, because the libInfoLogger recipe is built without Go bindings, and alisw/infrastructure-admins prefer to keep it that way.

Reply:
Actually you're right, I might be able to use libInfoLogger instead of InfoLogger at least in the OCCplugin.